tutorials/btp-integration-suite-design-guidelines/btp-integration-suite-design-guidelines.md

```diff
@@ -0,0 +1,139 @@
+
+2. Select the integration flow you're interested in by clicking on it. In our example, **Modeling Basics - Timer-Initiated Scenario**.
+
+    !![Modeling Basics](btw-design-guidelines_modeling-basics.png)  
+
+3. You are now looking at the integration flow editor view. Here, your integration flow is displayed, showcasing all its components and steps. Your integration flow consists of three components:
+
+    - A timer event **Start Timer** that is set to `Run Once` and that initiates the integration flow once it has been deployed;
+    - A message body **Define message body** that defines a hard-coded message body that reads `Hello World!`;
+    - A write step called **Write data store entry** that takes the message body and writes it to the data store.
+
+      !![Integration Flow](btw-design-guidelines_design-space-edit2.png)
+
+    > Tip: *Click on the respective shapes, to view the parameters of the respective integration flow step. You might need to pull up the window from the bottom of your screen.*
+
+    For integration flows, in general, you always have the following options:
+
+      - **Edit**: This option allows you to edit the components and steps of the integration flow. You can add or delete steps, and/or change the adapter setup. In general, the edit-option is unnecessary for integration flow design guidelines as these flows are constructed in such a way that they meet the required criteria already.
+
+    > **Caution**: In general, if you edit an integration flow, you will not receive any automatic updates for this edited integration flow once new features or steps are available.
+
+      - **Configure**: This option allows you to configure parameters more easily via a quickly accessible wizard without having to edit the entire integration flow.
+
+    In this tutorial, you do not need to configure or edit anything as this integration flow design guideline is ready to be deployed.
+
+
+3.  Choose **Deploy**.
+
+[DONE]
+[ACCORDION-END]
+
+
+[ACCORDION-BEGIN [Step 4: ](Check integration flow deployment)]
+
+In this final step, check whether the data store entry "Hello world" has been written to the data store.
+
+1. Wait for a message to tell you that the deployment was successful.
+
+2. From the left hand navigation bar, choose **Monitor**.
+
+3.  First, check the message processing log of the message. The message processing log (MPL) stores data about the messages processed on a tenant. Select **Monitor Message Processing** > **All Integration Flows**.
+
+    You now see a list of all messages that have run on this tenant. Select the entry with artifact name **Modeling Basics – Timer-Initiated Scenario** and check whether the Status displays a green success message: "Message processing completed successfully."
+
+4. Now, let's download the message. Go back to the **Monitor** overview and this time select **Manage Stores** > **Data Stores**.

```

Comment:
Same here, but with "us".
Better remove "let's" or say: "Now, it's time to download the message"

Reply:
Please make the suggested revisions and resubmit.